data/registry_model/registry_oci_model.py

```diff
@@ -1,36 +1,28 @@
 import logging
-from collections import defaultdict
 from contextlib import contextmanager
-
-from peewee import fn
+from typing import Optional
 
 from data import database, model
 from data.cache import cache_key
 from data.database import db_disallow_replica_use, db_transaction
-from data.model import DataModelException, oci
+from data.model import oci
 from data.model.oci.retriever import RepositoryContentRetriever
 from data.readreplica import ReadOnlyModeException
 from data.registry_model.datatype import FromDictionaryException
 from data.registry_model.datatypes import (
     Blob,
     BlobUpload,
     Label,
-    LegacyImage,
-    LikelyVulnerableTag,
     Manifest,
     ManifestLayer,
     RepositoryReference,
-    SecurityScanStatus,
     ShallowTag,
     Tag,
 )
 from data.registry_model.interface import RegistryDataInterface
 from data.registry_model.label_handlers import LABEL_EXPIRY_KEY, apply_label_to_manifest
 from data.registry_model.shared import SyntheticIDHandler
-from image.docker.schema1 import (
-    DOCKER_SCHEMA1_CONTENT_TYPES,
-    DockerSchema1ManifestBuilder,
-)
+from image.docker.schema1 import DOCKER_SCHEMA1_CONTENT_TYPES
```

Comment:
We still have plenty of Docker v2 schema 1 images stored in both Quay.io and people are still creating them locally. IMHO we cannot remove this support.